hnn_core/cell_response.py

```diff
@@ -0,0 +1,395 @@
+"""CellResponse class."""
+
+# Authors: Nick Tolley <nicholas_tolley@brown.edu>
+#          Ryan Thorpe <ryan_thorpe@brown.edu>
+#          Mainak Jas <mjas@mgh.harvard.edu>
```

Comment:
Surprised I noticed this, do you want to use the same email across files?

Reply:
I'll do this in a separate PR!